deploy/contents/install/app/install-utils.sh

```diff
@@ -562,6 +562,41 @@ function array_contains_or_empty () {
     return 1
 }
 
+function count_services {
```

Comment:
why you decide to name it this way? what is the main purpose of this function? For me it more about `enable` service than `count`